package.json

```diff
@@ -71,9 +72,9 @@
     "build": "npm run build-js && npm run build-css",
     "watch": "npm run build-example && npm run watch-example && (budo dist/example/main.js --live --css dist/example/main.css)",
     "lint": "eslint src",
-    "execute-tests": "run-browser-babel dist/test/index.js --phantom",
+    "execute-tests": "run-browser-babel dist/test/index.js --bx react/addons --bx react/lib/ReactContext --bx react/lib/ExecutionEnvironment --phantom",
```

Comment:
@chrisirhc , excluding from browserify. Otherwise, enzyme will be failing on compilation. The reason for it is simple: [this](https://github.com/airbnb/enzyme/blob/bd5249c8590874056ca63c5ba908de067c5b5e90/src/react-compat.js#L23) and other requires do not exist for react >=0.14.x, **but** are required during the browserify's compilation